great-idea/css/index.css

```diff
@@ -63,4 +65,227 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+*{
+	margin 10px;
+}
+
+body{
+	margin-left: 10px;
+	margin-right: 10px;
+}
+.nav{
+	display: flex;
+	justify-content: flex-start;
+
+}
+
+li {
```

Comment:
This may be the reason why your `li` in the service page are not in ideal spacing. Check out the Developer tools and see what you can come up with!